src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -34,12 +32,7 @@ public IntakeSubsystem() {
     intakeMotor.setSmartCurrentLimit(ManipulatorConstants.MOTOR_CURRENT_LIMIT);
     intakeMotor.setInverted(false);
     intakeMotor.setIdleMode(CANSparkFlex.IdleMode.kCoast);
-    intakeMotor.burnFlash(); 
-
-    // intakeMotorL.setInverted(false);
-    // intakeMotorR.setInverted(true);
-    //intakeMotorL.follow(intakeMotorR); // Sets the left motor to be the follow of the right intake motor
-    SmartDashboard.putData("IntakeSubsystem", this);
+    intakeMotor.burnFlash();
```

Comment:
I thought this caused problems? or was it just the reset motors and burn flash doesn't cause problems?